Typo in privilege string 'partner'

https://github.com/sourcetoad/tizen-build-action/blob/fb6685eff3762ee1bb172ee738e87fa4432a8ccd/build.sh#L17
Should be `partner`

Hi @iBotPeaches @chase-patterson, I assumed that this is a typo, therefore created this issue, because has problems with using distributor partner level certificate. Below i attach logs from my build:

```Installation has been completed!
Thank you for using Installer
/home/runner/work/_actions/sourcetoad/tizen-build-action/v1.1.0/build.sh: line 16: [partner: command not found
/home/runner/work/_actions/sourcetoad/tizen-build-action/v1.1.0/build.sh: line 43: [: too many arguments


Build Web Application:   10% ( 1/10)
Build Web Application:   20% ( 2/10)
Build Web Application:   40% ( 4/10)
Build Web Application:   50% ( 5/10)
Build Web Application:   60% ( 6/10)
Build Web Application:   70% ( 7/10)
Build Web Application:   80% ( 8/10)
Build Web Application:  100% (10/10)
Build Web Application:  100% (10/10)

BUILD SUCCESSFUL

Output path : /home/runner/work/DTC_Tizen/DTC_Tizen/build/.buildResult
Excluded : [.build/*, .build, .sign/*, .sign, webUnitTest/*, webUnitTest, .externalToolBuilders/*, .externalToolBuilders, .buildResult/*, .buildResult, .settings/*, .settings, .package/*, .package, .tproject, .project, .sdk_delta.info, .rds_delta, *.wgt, .tizen-ui-builder-tool.xml]
Total time: 00:00:00.582
An error has occurred. See the log file tizen-studio-data/cli/logs/cli.log.
2023-12-19 16:29:31,132 [TRACE] Main.java(133) - Start running Tizen CLI Main class...
2023-12-19 16:29:31,137 [TRACE] Main.java(134) - Argument count:4
2023-12-19 16:29:31,217 [TRACE] ConfigCLI.java(76) - Execute configCLI...
2023-12-19 16:29:32,678 [TRACE] Main.java(133) - Start running Tizen CLI Main class...
2023-12-19 16:29:32,685 [TRACE] Main.java(134) - Argument count:5
2023-12-19 16:29:32,711 [TRACE] BuildWebCLI.java(69) - Execute BuildWebCLI...
2023-12-19 16:29:32,718 [TRACE] BuildWebCLI.java(72) - getRealWorkingPath:/home/runner/work/DTC_Tizen/DTC_Tizen/build
2023-12-19 16:29:34,580 [TRACE] Main.java(133) - Start running Tizen CLI Main class...
2023-12-19 16:29:34,583 [TRACE] Main.java(134) - Argument count:11
2023-12-19 16:29:34,600 [TRACE] PackageCLI.java(1313) - Execute PackageCLI...
2023-12-19 16:29:34,600 [TRACE] PackageCLI.java(1315) - getRealWorkingPath: /home/runner/work/DTC_Tizen/DTC_Tizen/build/.buildResult
Error: -19 16:29:35,169 [ERROR] AbstractCLI.java(93) - org.tizen.common.sign.exception.CertificationException: Invaild password
java.lang.IllegalStateException: org.tizen.common.sign.exception.CertificationException: Invaild password
	at org.tizen.common.core.command.policy.UncaughtExceptionHandlingPolicy.uncaughtException(UncaughtExceptionHandlingPolicy.java:84)
	at org.tizen.common.core.command.Executor.execute(Executor.java:179)
	at org.tizen.common.core.command.Executor.execute(Executor.java:150)
	at org.tizen.ncli.subcommands.sign.SignCLICommand.getSigningProfile(SignCLICommand.java:139)
	at org.tizen.ncli.subcommands.sign.SignCLICommand.call(SignCLICommand.java:73)
	at org.tizen.ncli.subcommands.sign.SignCLICommand.call(SignCLICommand.java:58)
	at org.tizen.ncli.subcommands.AbstractSubCommand.runCommand(AbstractSubCommand.java:76)
	at org.tizen.ncli.subcommands.packages.PackageCLICommand.call(PackageCLICommand.java:177)
	at org.tizen.ncli.subcommands.packages.PackageCLICommand.call(PackageCLICommand.java:77)
	at org.tizen.ncli.subcommands.AbstractSubCommand.runCommand(AbstractSubCommand.java:76)
	at org.tizen.ncli.ide.shell.PackageCLI.doPackageForWgt(PackageCLI.java:968)
	at org.tizen.ncli.ide.shell.PackageCLI.execute(PackageCLI.java:1450)
	at org.tizen.ncli.ide.shell.AbstractCLI.execute(AbstractCLI.java:91)
	at org.tizen.ncli.ide.shell.Main.run(Main.java:189)
	at org.tizen.ncli.ide.shell.Main.main(Main.java:122)
Caused by: org.tizen.common.sign.exception.CertificationException: Invaild password
	at org.tizen.common.sign.command.ReadSigningProfileFileCommand.checkPkcs12Password(ReadSigningProfileFileCommand.java:131)
	at org.tizen.common.sign.command.ReadSigningProfileFileCommand.validateSigningInfo(ReadSigningProfileFileCommand.java:146)
	at org.tizen.common.sign.command.ReadSigningProfileFileCommand.run(ReadSigningProfileFileCommand.java:101)
	at org.tizen.common.core.command.Executor.execute(Executor.java:164)
	... 13 more
Error: -19 16:29:35,172 [ERROR] Main.java(197) - org.tizen.common.sign.exception.CertificationException: Invaild password
web-app clean success.
Error: Process completed with exit code 1.
```
I'm pretty sure that distributor key and password are correct. Thank you for your help in advance

--- a/build.sh
+++ b/build.sh
@@ -13,7 +13,7 @@ PATH="$TIZEN_STUDIO/tools/ide/bin:$PATH"
 #
 # Parse arguments
 #
-if [$8 -eq 'partner']; then
+if [ "$8" = "partner" ]; then
     PRIVILEGE=parner
 else
     PRIVILEGE=public

